src/setupTests.ts

```diff
@@ -0,0 +1,14 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
file names shouldn't be in kebab-case? you have several camel case

Reply:
this file is looked by JEST, sure might be renamed, but for convinience not worth